server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/step/LoadReportAnalysisMetadataHolderStep.java

```diff
@@ -112,10 +113,13 @@ private void loadProject(ScannerReport.Metadata reportMetadata, Organization org
   }
 
   private Organization loadOrganization(ScannerReport.Metadata reportMetadata) {
-    Organization organization = toOrganization(ceTask.getOrganizationUuid());
-    checkOrganizationKeyConsistency(reportMetadata, organization);
-    analysisMetadata.setOrganization(organization);
-    return organization;
+    try (DbSession dbSession = dbClient.openSession(false)) {
+      analysisMetadata.setOrganizationsEnabled(organizationFlags.isEnabled(dbSession));
```

Comment:
minor: I would not call `organizationFlags.isEnabled` before `checkOrganizationKeyConsistency`, if the check fails, we uselessly made a call to DB